### DISCUSSION TITLE
Wiki Extensions category suggestion

### DISCUSSION THREAD

**Discussion by w-e-w** [camenduru/sd-webui-extension-records#4] [General]
I'm currently in the process of categorizing the Extensions and Scripts for the Wiki
I have made a initial list of extensions and there categories 
see [Wiki Extensions page categorization suggestion](https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/7665#discussion-4836938)
any feedback would be appreciated

**Comment by Harvester62** [comment]
I've seen your categorization of Scripts and Extensions in the SDWUI Wiki and I find it quite useful. I was thinking if the general lists for both Scripts and Extensions in the SDWUI Wiki wouldn't be easier to search by scrolling it if they would be in alphabetical order instead. 

Anyway, having also the list in this repository (the readme file) categorized in the same way I think might make sense, at least it would match the same criteria used in the SDWUI Wiki. Another possible way to list the working Extenstions might be by "last date verified", as an alternate view of the list, or even better by date of the SDWUI hash/commit. Just thinking loudly.

**Comment by TFWol** [comment]
A simple table would be nice, sortable columns and you're good to go. 
Problem is it's not possible without hosting the page since something like JavaScript would be needed.

I rely on a [userscript](https://github.com/Mottie/GitHub-userscripts/blob/master/github-sort-content.user.js) for that though. Lifesaver.

**Reply by Harvester62** [reply]
I do understand, and yes, as you say, a simple table with sortable columns would be enough.
